opencti-platform/opencti-graphql/src/schema/fieldDataAdapter.ts

```diff
@@ -173,6 +173,8 @@ export const booleanAttributes = [
   'aslr_enabled',
   'dep_enabled',
   'otp_mandatory',
+  'public',
```

Comment:
maybe attributes must be stream_public and stream_live to prevent future potential collision?

Reply:
Done